Bookish.DataAccess/BookishService.cs

```diff
@@ -87,5 +90,39 @@ ORDER BY
 
             return dbConnection.Query<BookLoan>(query, new { UserID = userId });
         }
+
+        public void AddBook(string title, string author, string isbn, int totalCopies)
+        {
+            var lastId = LastCopiesId();
+
+            var valuesList = new List<string>();
```

Comment:
I think you can use [Enumerable.Repeat](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.repeat?redirectedfrom=MSDN&view=netcore-3.1#System_Linq_Enumerable_Repeat__1___0_System_Int32_) to make this a bit easier 🙂